cypress/integration/step_definitions/backend/at_generate_token.cy.js

```diff
@@ -0,0 +1,25 @@
+import { Given, When } from "cypress-cucumber-preprocessor/steps";
+
+Given('User should get refresh_token and create authToken', ()=>{
+
```

Comment:
No need to add this function one more time. We already have it and it will find and run that function